libs/components/editor-blocks/src/blocks/code/CodeView.tsx

```diff
@@ -187,7 +200,7 @@ export const CodeView = ({ block, editor }: CreateCodeView) => {
                     </div>
                     <div>
                         <div className="copy-block" onClick={copyCode}>
-                            Copy
+                            <DuplicateIcon></DuplicateIcon>Copy
```

Comment:
self-closing.

Have you properly configured the ESLint?

<img width="640" alt="Screen Shot 2022-08-12 at 3 25 38 PM" src="https://user-images.githubusercontent.com/18554747/184305593-01643a61-896a-4204-8f12-527385a9f1bc.png">